lntest/node.go

```diff
@@ -248,6 +249,10 @@ type HarnessNode struct {
 	lnrpc.WalletUnlockerClient
 
 	invoicesrpc.InvoicesClient
+
+	// RouterClient cannot be embedded, because a name collision would occur
+	// on the main rpc SendPayment.
+	RouterClient routerrpc.RouterClient
```

Comment:
We have the same problem on mobile actually, it creates an API conflict if it is compiled with the router subserver. Should we consider changing the API name? Especially if we want to to enable it by default at some point.

Reply:
Maybe, but definitely not in this PR :)